src/main/java/com/example/springboot/service/CsvParserService.java

```diff
@@ -0,0 +1,7 @@
+package com.example.springboot.service;
+
+import com.example.springboot.dto.ReviewDto;
+
+public interface CsvParserService {
+    ReviewDto parseLine(String str);
```

Comment:
```suggestion
public interface CsvParserService<T> {
    T parseLine(String str);
```

let's better parameterise this service